app/scripts/lib/transaction/metrics.ts

```diff
@@ -996,11 +996,13 @@ async function buildEventFragmentProperties({
   if (simulationFails) {
     uiCustomizations.push(MetaMetricsEventUiCustomization.GasEstimationFailed);
   }
+
   const isRedesignedConfirmationsDeveloperSettingEnabled =
-    transactionMetricsRequest.getIsRedesignedConfirmationsDeveloperEnabled() ||
-    Boolean(process.env.ENABLE_CONFIRMATION_REDESIGN);
+    Boolean(process.env.ENABLE_CONFIRMATION_REDESIGN) &&
+    transactionMetricsRequest.getIsRedesignedConfirmationsDeveloperEnabled();
 
   const isRedesignedTransactionsUserSettingEnabled =
+    Boolean(process.env.ENABLE_CONFIRMATION_REDESIGN) &&
```

Comment:
We cannot do this... `ENABLE_CONFIRMATION_REDESIGN` is just intended for local dev. (we should get rid of it as soon as possible to avoid confusion)